frontend/talentsearch/src/js/components/signAndSubmit/SignAndSubmitPage.tsx

```diff
@@ -0,0 +1,284 @@
+import * as React from "react";
+import { Link } from "@common/components";
+import { Input, Submit } from "@common/components/form";
+import NotFound from "@common/components/NotFound";
+import Pending from "@common/components/Pending";
+import TableOfContents from "@common/components/TableOfContents";
+import { commonMessages, errorMessages } from "@common/messages";
+import { ArrowSmRightIcon } from "@heroicons/react/outline";
+import {
+  BriefcaseIcon,
+  ClipboardCheckIcon,
+  UserIcon,
+} from "@heroicons/react/solid";
+import { uniqueId } from "lodash";
+import { FormProvider, useForm } from "react-hook-form";
+import { useIntl } from "react-intl";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+import ApplicationPageWrapper from "../ApplicationPageWrapper/ApplicationPageWrapper";
+
+const ImportantInfo = () => {
+  const intl = useIntl();
+  const steps = [
+    intl.formatMessage({
+      defaultMessage:
+        "When you submit your application, a copy of your profile will be created.",
+      description: "Important info list item on sign and submit page.",
+    }),
+    intl.formatMessage({
+      defaultMessage:
+        "This copy will be used for the initial application review.",
+      description: "Important info list item on sign and submit page.",
+    }),
+    intl.formatMessage({
+      defaultMessage:
+        "Changes made to your profile after submitting will not be updated on this copy.",
+      description: "Important info list item on sign and submit page.",
+    }),
+    intl.formatMessage({
+      defaultMessage:
+        "You are still encouraged to keep your profile up to date, updated versions will be used at later steps of the hiring process.",
+      description: "Important info list item on sign and submit page.",
+    }),
+  ];
+
+  return (
+    <ol>
+      {steps.map((item) => (
+        <li key={uniqueId()} data-h2-margin="b(bottom, m)">
+          {item}
+        </li>
+      ))}
+    </ol>
+  );
+};
+
+const Signature: React.FC<{ isNotComplete: boolean }> = ({ isNotComplete }) => {
```

Comment:
See: The other comment about `React.FC`

Reply:
Fixed by [Remove React.FC type from components](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3678/commits/c8632abf5980eac17565ab2f878ec1498f93d15e)